lib/reporting/reports/orders_and_fulfillment/order_cycle_supplier_totals.rb

```diff
@@ -27,14 +27,14 @@ def rules
               summary_row: proc do |_key, _items, rows|
                 total_units = rows.map(&:total_units)
                 summary_total_units = if total_units.all?(&:present?)
-                                        rows.map(&:total_units).sum(&:to_f)
+                                        rows.map(&:total_units).sum(&:to_f).round(3)
```

Comment:
Same here, units can actually be kilograms, a test is covering the non-integer units case.